src/Analysis/Ast/Impl/Types/PythonClassType.cs

```diff
@@ -30,10 +30,10 @@
 
 namespace Microsoft.Python.Analysis.Types {
     [DebuggerDisplay("Class {Name}")]
-    internal partial class PythonClassType : PythonType, IPythonClassType, IGenericType, IEquatable<IPythonClassType> {
+    internal partial class PythonClassType {
```

Comment:
A nit, but I think that it should get removed from `PythonClassType.Generics.cs` rather than `PythonClassType.cs`, if anywhere.